MOBILE_API.xml

```diff
@@ -647,44 +647,106 @@
         <element name="CUSTOM_BUTTON" />
         <element name="SEARCH" />
         <!-- Climate Buttons -->
-        <element name="AC_MAX" since="4.5" />
-        <element name="AC" since="4.5" />
-        <element name="RECIRCULATE" since="4.5" />
-        <element name="FAN_UP" since="4.5" />
-        <element name="FAN_DOWN" since="4.5" />
-        <element name="TEMP_UP" since="4.5" />
-        <element name="TEMP_DOWN" since="4.5" />
-        <element name="DEFROST_MAX" since="4.5" />
-        <element name="DEFROST" since="4.5" />
-        <element name="DEFROST_REAR" since="4.5" />
-        <element name="UPPER_VENT" since="4.5" />
-        <element name="LOWER_VENT" since="4.5" />
+        <element name="AC_MAX" since="4.5">
+            <description>Tied to CLIMATE modules.</description>
```

Comment:
```suggestion
            <description>Tied to CLIMATE RC modules.</description>
```

If correct, this would affect all the button changes.

Reply:
Fixed in https://github.com/smartdevicelink/rpc_spec/pull/290/commits/bdbb236b620e46efb451cf4204aa0c955f9598b6